catalyst/core/callbacks/optimizer.py

```diff
@@ -7,7 +7,7 @@
 
 from catalyst import utils
 from catalyst.core import _State, Callback, CallbackOrder, registry
-from catalyst.utils.typing import Optimizer
+from catalyst.utils.tools.typing import Optimizer
```

Comment:
but `typing` is not a tool

Reply:
i suggest moving typing in new `catalyst.typing` module :)
note: not in `catalyst.utils`

Reply:
or, could you create `catalyst.typing` package with something like

```python
# __init__.py for catalyst.typing
from catalyst.utils.tools.typing import *
```

and in the PR just use `from catalyst.typing import Optimizer`. i think it's much clearer